packages/scene-composer/src/hooks/useKeyPress.ts

```diff
@@ -0,0 +1,92 @@
+/* istanbul ignore file */
+// Credit original Source: https://codesandbox.io/s/rob7b8?file=/src/hooks/useKeyPress.tsx:0-2069
+// Heavily modified for my needs
```

Comment:
Nit-pick: Probably don't want to keep this line as-is in the comments. Suggest clarifying what "my needs" are instead.

Reply:
Removed